File: front-end/src/app/endpoints/kompanija-endpoint/update/kompanija-update-request.ts
export interface KompanijaUpdateRequest {
  id: string,
  naziv: string | null,
  lokacija: string | null,
  brojZaposlenih: string | null,
  website: string | null,
  linkedIn: string | null,
  twitter: string | null,
  kratkiOpis: string | null,
  opis: string | null,
  logo: string | ArrayBuffer | null
}

File: JobSearchingWebApp/Endpoints/Kompanija/Update/KompanijaUpdateRequest.cs
using System.ComponentModel.DataAnnotations;

namespace JobSearchingWebApp.Endpoints.Kompanija.Update
{
    public class KompanijaUpdateRequest
    {
        [Required]
        public string Id {  get; set; }  

        public string? Naziv { get; set; }

        public string? Lokacija { get; set; }

        public string? BrojZaposlenih { get; set; }

        public string? Logo{ get; set; }

        public string? Website { get; set; }

        public string? LinkedIn { get; set; }

        public string? Twitter { get; set; }

        public string? KratkiOpis { get; set; }

        public string? Opis { get; set; }
    }
}
